packages/sanes-browser-extension/src/routes/tx-request/components/ProposalActionType/SetValidators.tsx

```diff
@@ -0,0 +1,59 @@
+import { SetValidatorsAction, ValidatorProperties } from "@iov/bns";
+import { ListItem, ListItemText } from "medulas-react-components";
+import * as React from "react";
+
+import {
+  txListItemSecondaryProps,
+  useTxListItemHeaderStyles,
+  useTxListItemStyles,
+} from "../ShowRequest/TransactionFee";
+
+interface ValidatorProps {
+  readonly index: string;
+  readonly properties: ValidatorProperties;
+}
+
+function ValidatorsListItem({ index, properties }: ValidatorProps): JSX.Element {
+  const listItemClasses = useTxListItemStyles();
+  const validatorLabel = `${index.slice(8, 28)}...${index.slice(-3)}`;
```

Comment:
Can you create an `ellipsifyMiddle` method in `ui-logic` / `strings.ts`  that ellipsifies the pubkey (`= index.slice(8)`) with the same logic you use here?